### PR TITLE
SNOW-925990 add logging for mapping resultset columns

### PR DESCRIPTION
**Pull request by sfc-gh-dszmolka**
### Description
* This is a very small logging change required earlier by Support, for having some insight into the event of mapping resultset columns.
* Requirements and performance test on tw676.

### Checklist
- [ ] Format code according to the existing code style (run `npm run lint:check -- CHANGED_FILES` and fix problems in changed code)
- [ ] Create tests which fail without the change (if possible)
- [ ] Make all tests (unit and integration) pass (`npm run test:unit` and `npm run test:integration`)
- [ ] Extend the README / documentation and ensure is properly displayed (if necessary)
- [x] Provide JIRA issue id (if possible) or GitHub issue id in commit message

### CODE REVIEW

File: lib/connection/result/result.js [hunk 1]
```diff
@@ -113,6 +114,7 @@ function Result(options) {
       mapColumnNameToIndices[column.getName()] || [];
     mapColumnNameToIndices[column.getName()].push(index);
   }
+  Logger.getInstance().trace(`Finished mapping columns.`);
```
Comment (sfc-gh-pmotacki) [inline lib/connection/result/result.js:117]: It's better to use singlequotes for strings without vars.

File: lib/connection/result/result.js [hunk 2]
```diff
@@ -101,6 +101,7 @@ function Result(options) {
     ColumnNamesCreator.addOverridenNamesForDuplicatedColumns(rowtype);
   }
 
+  Logger.getInstance().trace(`Mapping columns in resultset (total: ${numColumns})`);
```
Comment (sfc-gh-pmotacki) [inline lib/connection/result/result.js:104]: We are only interested in quantity?

File: lib/connection/result/result.js (hunk 1 above)
Comment (sfc-gh-dszmolka) [reply lib/connection/result/result.js]: fixed

File: lib/connection/result/result.js (hunk 2 above)
Comment (sfc-gh-dszmolka) [reply lib/connection/result/result.js:104]: for now; yes. the request was to somehow log column mapping progress

**Review by sfc-gh-pmotacki — APPROVED** [review]
LGTM

**Pull request merged**